### ISSUE TITLE
refresh networking build flags in all Wiki articles (Bugzilla Bug 1357)

### ISSUE THREAD

**Issue by tianocore-issues** [tianocore/tianocore.github.io#35]
This issue was created automatically with [bugzilla2github](https://github.com/makubacki/bugzilla2github/tree/bz2gh_tianocore)
# Bugzilla Bug 1357
Date: 2018-11-23T06:54:29+00:00
From: @lersek
To:   sfu5 &lt;<siyuan.fu>&gt;
CC:   @lersek, siyuan.fu, yonghong.zhu

Depends on:   #1293
Last updated: 2020-05-05T19:07:31+00:00

**Comment by tianocore-issues** [comment]
## Comment 5698
Date: 2018-11-23 06:54:29 +0000
From: @lersek

- Industry Specification: ---
- Releases to Fix: EDK II Master
- Target OS: ---
- Bugzilla Assignee(s): sfu5 &lt;<siyuan.fu>&gt;

Under bug 1293, NetworkPkg is gaining include files (DSC and FDF file snippets), which depend on a number of build-time &quot;-D&quot; flags. The flags are also owned / defined by NetworkPkg, and they replace platform build flags.

For example, &quot;HTTP_BOOT_ENABLE&quot; in OvmfPkg is being replaced by the central &quot;NETWORK_HTTP_BOOT_ENABLE&quot;.

For another example, PcdAllowHttpConnections should now be set via the NETWORK_ALLOW_HTTP_CONNECTIONS build flag.

The patch series for bug 1293 handles all this in the edk2 source tree, but the Wiki articles should be searched &amp; updated too. Hence this separate BZ report. Thanks.

**Comment by tianocore-issues** [comment]
## Comment 5773
Date: 2018-12-06 19:14:57 +0000
From: Yonghong Zhu &lt;<yonghong.zhu>&gt;

Bug scrub: Assign to module owner

**Comment by tianocore-issues** [comment]
## Comment 12387
Date: 2020-05-05 19:07:31 +0000
From: @lersek

Nobody had capacity for 1.5 years, closing as WONTFIX.
